src/Tutorials/Minimal-Orleans-Application.md

```diff
@@ -8,20 +8,20 @@ title: Minimal Orleans Application
 This tutorial provides step by step instructions for creating a basic functioning Orleans application. It is designed to be self-contained and minimalistic, with the following traits.
 
 
-- It does not need the SDK or Extension to be installed, and relies on the Nuget packages only.
-- Tested in both Visual Studio 2013 and 2015 using Orleans 1.2.0.
+- It does not need the SDK or Extension to be installed, and relies on Nuget packages only.
+- Has been tested in both Visual Studio 2013 and 2015 using Orleans 1.2.0.
 - Has no reliance on SQL or Azure
 
-Keep in mind, this is only a tutorial and lacks appropriate error handling and other goodies that would be useful for a production environment. However, it should help the reader get a real hands-on for all the different facets of Orleans and allow them to focus their continued learning on the parts most relevant to them.
+Keep in mind that this is only a tutorial and lacks appropriate error handling and other goodies that would be useful for a production environment. However, it should help the readers get a real hands-on with regards to different facets of Orleans and allow them to focus their continued learning on the parts most relevant to them.
 
 You can find the final source code in `Samples\Tutorial.Minimal` folder ([see latest version online](https://github.com/dotnet/orleans/tree/master/Samples/Tutorial.Minimal)).
 
 ## Project Setup
 
 
-For this tutorial we’re going to need to create 3 projects.  A Library that contains the interfaces (Communication interfaces), a library that contains the implementation (called Grains), and a simple console application that will Host our Silo.  For further understanding of this terminology please see the document Getting Started with Orleans.
+For this tutorial we’re going to need to create 3 projects.  A Library that contains the interfaces (Communication interfaces), a library that contains the implementation (called Grains), and a simple console application that will Host our Silo.  For further understanding of this terminology please see the document [Getting Started with Orleans](https://dotnet.github.io/orleans/Tutorials/My-First-Orleans-Application.html).
```

Comment:
you should be able to add a relative link to the markdown file instead, which will then be converted correctly depending on where the docs are hosted.
In this case it should be `[Getting Started with Orleans](My-First-Orleans-Application.md)